front/src/modules/simulationResult/components/SpaceTimeChart/ManchetteMenuButton.tsx

```diff
@@ -15,6 +17,19 @@ const SpaceTimeChartHeader = () => {
 
   const closeMenu = () => setIsMenuOpen(false);
 
+  const menuItems: OSRDMenuItem[] = useMemo(
+    () => [
+      {
+        title: t('manchetteSettings.waypointsVisibility'),
+        icon: <Eye />,
+        onClick: () => {
+          closeMenu(); // TODO : in #8628, change this to open the waypoints modal
+        },
+      },
+    ],
+    [t]
+  );
```

Comment:
do we really need to memoize?

Reply:
You're right, it's probably overkill

Reply:
I quite agree with @Yohh , I don't think `useMemo` is necessary here. The `menuItems` array is relatively simple and contains elment witth simple property. Removing `useMemo` would simplify the code without significantly impacting performance.